.github/workflows/stale.yml

```diff
@@ -0,0 +1,28 @@
+name: "Close stale issues and PRs"
+on:
+  schedule:
+    - cron:  '00 05 * * *'
+
+jobs:
+  stale:
+    permissions:
+      issues: write
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: gorzelinski/stale@add-reaction-updates
+        with:
+          repo-token: ${{ secrets.GH_BOT_TOKEN }}
+          stale-issue-label: status:stale
+          close-issue-label: resolution:expired
+          stale-issue-message: "There has been no activity on this issue for the last five years. We have marked it as stale and will close it in 30 days. We know it may be valid, so if you're interested in the solution, leave a comment or reaction under this issue."
+          stale-pr-message: "There has been no activity on this PR for the last five years. We have marked it as stale and will close it in 30 days. We know it may be valid, so if you're interested in the contribution, leave a comment or reaction under this PR."
+          close-issue-message: "We have closed the issue due to the lack of activity for the last five years. We know it may be valid, so you can reopen a similar issue."
+          close-pr-message: "We have closed the PR due to the lack of activity for the last five years. We know it may be valid, so you can reopen a similar PR."
```

Comment:
```suggestion
          close-pr-message: "We've closed your PR due to inactivity over the past five years. While time has passed, the core of your contribution might still be relevant. If you're open to it, consider reopening a similar PR."
```

Reply:
"Feel free..." - it's funny because I used the exact wording before committing. But eventually, I decided it was a little wordy. On the other hand, it sounded friendly. Emphising "you" instead of "we" is a good idea. I'll make the changes in both repositories.